BlargBucket/ViewControllers/RepositoryInfoViewController.swift

```diff
@@ -79,5 +79,13 @@ class RepositoryInfoViewController: UITableViewController {
 		cell.detailTextLabel!.text = model.detailTitle
 		return cell
 	}
+	
+	override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+		tableView.deselectRowAtIndexPath(indexPath, animated: true)
+		let model = indexPath.section == 0 ? viewModel.info![indexPath.row] : viewModel.standardOptions[indexPath.row]
```

Comment:
We can go with that. It makes me seem slightly less lazy than just copying the one that already existed.